Makefile

```diff
@@ -136,6 +136,7 @@ test-unit:
 # Build and vendor tarets
 #---------------------------------------------------------
 
+.PHONY: ./vendor
```

Comment:
The `./vendor` need not be a PHONY target. It's an output directory. See the introduction: https://www.gnu.org/software/make/manual/html_node/Rule-Introduction.html#Rule-Introduction

Reply:
Thanks @baijum I'll remove it.